base/strings/search.jl

```diff
@@ -101,8 +101,23 @@ julia> findfirst("Julia", "JuliaLang")
 1:5
 ```
 """
-findfirst(pattern::AbstractString, string::AbstractString) =
-    findnext(pattern, string, firstindex(string))
+findfirst(pattern::AbstractString, string::AbstractString) = findnext(pattern, string, firstindex(string))
+
+"""
+    findfirst(ch::Char, string::AbstractString)
+
+Find the first occurrence of `ch` in `string`. Equivalent to
+[`findfirst(isequal(ch), string)`](@ref).
+
+# Examples
+```jldoctest
+julia> findfirst('a', "happy")
+2
+
+julia> findfirst('z', "happy") # returns nothing, but not printed in the REPL
```

Comment:
This might be slightly clearer:
```suggestion
julia> findfirst('z', "happy") === nothing
true
```

Same below.